vnext/ReactUWP/Views/ViewManagerBase.cpp

```diff
@@ -229,6 +235,14 @@ void ViewManagerBase::UpdateProperties(ShadowNodeBase* nodeToUpdate, const dynam
     {
       nodeToUpdate->m_onLayout = !propertyValue.isNull() && propertyValue.asBool();
     }
+    else if (propertyName == "keyDownEvents")
```

Comment:
>else if (propertyName == "keyDownEvents") [](start = 4, length = 41)

Did we want key events on all native types or only <View> ?  this puts it on everything where most other input stuff is only on ViewViewManager

Reply:
On controls more than view.

---
In reply to: [295393138](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295393138) [](ancestors = 295393138)

Reply:
Just want to make sure - is it better in here or in FrameworkElementViewManager?

---
In reply to: [295444711](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295444711) [](ancestors = 295444711,295393138)

Reply:
KeyUp/KeyDown event is added in this class in GetExportedCustomBubblingEventTypeConstants. I'd like to keep it in ViewBasemanager too.

---
In reply to: [295570563](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295570563) [](ancestors = 295570563,295444711,295393138)